packages/orbit-components/src/Itinerary/ItinerarySegment/ItinerarySegmentStop/index.jsx

```diff
@@ -0,0 +1,74 @@
+// @flow
+import * as React from "react";
+import styled, { css } from "styled-components";
+
+import { useWidth } from "../../context";
+import defaultTheme from "../../../defaultTheme";
+import Stack from "../../../Stack";
+import Text from "../../../Text";
+import ItineraryIcon from "../ItineraryIcon";
+
+import type { Props } from ".";
+
+const StyledWrapper = styled.div`
+  ${({ theme, $hidden }) => css`
+    display: flex;
+    position: relative;
+    box-sizing: border-box;
+    opacity: ${$hidden ? `0.8` : `1`};
+    padding: 0 ${theme.orbit.spaceSmall};
+  `}
+`;
+
+// $FlowFixMe: https://github.com/flow-typed/flow-typed/issues/3653#issuecomment-568539198
+StyledWrapper.defaultProps = {
+  theme: defaultTheme,
+};
+
+const StyledDate = styled.div`
+  ${({ minWidth }) => css`
+    min-width: ${minWidth}px;
+  `}
+`;
+
+const ItinerarySegmentStop = ({
+  date,
+  icon,
+  time,
+  city,
+  station,
+  hidden,
+  minWidth = 60,
+  type,
+}: Props): React.Node => {
+  const { calculatedWidth, setWidths } = useWidth();
+  const dateRef = React.useRef<HTMLDivElement | null>(null);
+
+  React.useEffect(() => {
+    setWidths(prev => (dateRef.current ? [...prev, dateRef.current.clientWidth] : prev));
+  }, [setWidths]);
+
+  return (
+    <StyledWrapper $hidden={hidden}>
+      <Stack inline align="center" spacing="small">
+        <StyledDate minWidth={calculatedWidth || minWidth} ref={dateRef}>
+          <Stack flex direction="column" spacing="XXXSmall" align="end">
+            <Text weight="bold">{time}</Text>
```

Comment:
Let's wait until design review, ok? Because even in the design colors do not match, for instance, we have different `inkLight` color than in figma, because of [this](https://jira.kiwi.com/browse/ORBIT-1820) and i think it should be updated in the design.